### DISCUSSION TITLE
Close, Max and Min buttons not appearing

### DISCUSSION THREAD

**Discussion by Eloque** [TomSchimansky/CustomTkinter#2358] [Q&A]
So, I am running the example code, and my window looks like this:

![image](https://github.com/TomSchimansky/CustomTkinter/assets/4184679/872eb90d-abe9-4950-83e9-de34c5356157)

As you can see the buttons are gone, I have no idea why. Didn't use to do that. On Python 3.10.12, with wxPython 4.2.1. I tried recreating the venv and installing everything from scratch. Still happens.

Is this likely a bug, or something wrong on my end?

**Comment by dipeshSam** [comment]
@Eloque **Which operating system are you on?**

Please mention clear and full photo, it seems a bit edited or cropped.

**Reply by Eloque** [reply]
The reason it seems like that, is because I can't move the window. Here is another one, with the IDE behind it. Nothing edited or cropped.

![image](https://github.com/TomSchimansky/CustomTkinter/assets/4184679/61d07681-f39a-434c-90e1-129e58893d18)

I am on Windows 10 Pro, but run PyCharm on WSL, so essentially Ubuntu 22.04.4 LTS on 5.15.146-1
Here is a pip list for good measure:

```
pip list
Package            Version
------------------ -----------
Babel              2.14.0
certifi            2024.2.2
charset-normalizer 3.3.2
contourpy          1.2.1
customtkinter      5.2.2
cycler             0.12.1
darkdetect         0.8.0
docutils           0.20.1
easygui            0.98.3
fonttools          4.51.0
idna               3.6
Kivy               2.3.0
Kivy-Garden        0.1.5
kiwisolver         1.4.5
matplotlib         3.8.4
numpy              1.26.4
packaging          24.0
pillow             10.3.0
pip                24.0
psutil             5.9.8
Pygments           2.17.2
pyparsing          3.1.2
python-dateutil    2.9.0.post0
requests           2.31.0
setuptools         68.2.0
shapely            2.0.3
six                1.16.0
svgelements        1.9.6
tk                 0.1.0
ttkthemes          3.2.2
urllib3            2.2.1
wheel              0.41.2
wxPython           4.2.1

```

**Reply by dipeshSam** [reply]
The issue is most probably due to mismatch High DPI Awareness in Windows Pro.

A quick solution could be disabling the DPI Awareness manually:

``` Python
from customtkinter import deactivate_automatic_dpi_awareness

deactivate_automatic_dpi_awareness()

if __name__ == "__main__":
        ...
```

Note: Use this statement before the program starts.

**Reply by Eloque** [reply]
I've tried it, but it didn't change anything. Windows 10 is the host system I am running Windows Subsystem for Linux, does the dpi awareness transfer over into WSL?

The test code is now this:

```
from customtkinter import deactivate_automatic_dpi_awareness
deactivate_automatic_dpi_awareness()

import customtkinter

customtkinter.set_appearance_mode("System")  # Modes: system (default), light, dark
customtkinter.set_default_color_theme("blue")  # Themes: blue (default), dark-blue, green

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
app.geometry("400x240")

def button_function():
    print("button pressed")

# Use CTkButton instead of tkinter Button
button = customtkinter.CTkButton(master=app, text="CTkButton", command=button_function)
button.place(relx=0.5, rely=0.5, anchor=customtkinter.CENTER)

app.mainloop()
```

**Reply by dipeshSam** [reply]
Following should work:

``` Python

import customtkinter

customtkinter.set_appearance_mode("System")  # Modes: system (default), light, dark
customtkinter.set_default_color_theme("blue")  # Themes: blue (default), dark-blue, green

customtkinter.deactivate_automatic_dpi_awareness()

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
app.geometry("400x240")

def button_function():
    print("button pressed")

# Use CTkButton instead of tkinter Button
button = customtkinter.CTkButton(master=app, text="CTkButton", command=button_function)
button.place(relx=0.5, rely=0.5, anchor=customtkinter.CENTER)

app.mainloop()

```

**Reply by Eloque** [reply]
Unfortunately, it doesn't

![image](https://github.com/TomSchimansky/CustomTkinter/assets/4184679/938dd763-8d07-4393-89a3-1994c1e434e7)

Seems like the whole border is transparant. Can't move the window either, so that's why the weird screenshots.

**Reply by Akascape** [reply]
@Eloque I can suggest you to try adding these methods:

`root.resizable(True, True)`

```
root.wm_attributes('-fullscreen', 1)
root.wm_attributes('-fullscreen', 0)
```


```
root.overrideredirect(1)
root.overrideredirect(0)
```

```
root.update()
root.update_idletasks()
```

Also let me know of this issue is happening with tkinter?

**Reply by Eloque** [reply]
The suggested code does not help.

But, it does also happen on bare tkinter. So it seems logical that the problem is not with Ctk.

**Reply by Akascape** [reply]
@Eloque Probably an issue with the environment you are using. I recommend you to change the environment or use this window method to remove the title bar: 
https://github.com/TomSchimansky/CustomTkinter/discussions/1756#discussioncomment-6274965

**Reply by Eloque** [reply]
I am pulling hairs. I've recreated the complete virtual env so far. Will try more rigorous things later.

**Comment by avalon60** [comment]
Where did you obtain Python from? If it is not from Python.org, I would suggest that you reinstall from there.
